packages/driver/src/cypress/ensure.ts

```diff
@@ -255,7 +255,7 @@ const commandCanCommunicateWithAUT = (cy: $Cy, err?): boolean => {
     const crossOriginCommandError = $errUtils.errByPath('miscellaneous.cross_origin_command', {
       commandOrigin: window.location.origin,
       autOrigin: cy.state('autLocation').origin,
-      isSkipDomainInjectionEnabled: !!Cypress.config('experimentalSkipDomainInjection'),
+      isInjectDocumentDomainEnabled: !!Cypress.config('injectDocumentDomain'),
```

Comment:
is there a time where this `undefined`?

Reply:
Shouldn't be. Will address.

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/30770/commits/6c52c935ac1c34867ca4a04a93aad38b1801f485